lab_2_tokenize_by_bpe/main.py

```diff
@@ -58,6 +115,30 @@ def train(
     :param num_merges: required number of new tokens
     :return: dictionary in the form of <preprocessed word: number of occurrences>
     """
+    if not isinstance(word_frequencies, dict) or not isinstance(num_merges, int):
+        return None
+    while num_merges > 0:
```

Comment:
Are you sure that this condition is enough?

Reply:
fixed